tests/integration/league_of_legends/test_LeagueApiV4.py

```diff
@@ -82,6 +85,17 @@ def test_by_summoner(self, lol_context, region, encrypted_summoner_id):
             actual_response,
         )
 
+    @pytest.mark.parametrize("puuid", ["dQw4w9WgXcQ", "12093qowie"])
+    def test_by_summoner(self, lol_context, region, puuid):
```

Comment:
rename `test_by_puuid` - as written this is just shadowing the existing `test_by_summoner` function